python/addons/custom_vectorizers.py

```diff
@@ -81,6 +81,8 @@ def __getstate__(self):
         return {'codes': self.codes, 'vocab': self.vocab}
 
     def __setstate__(self, state):
+        from fastBPE import fastBPE
+        import tempfile
```

Comment:
tempfile is a std library import so it should go at the top of the file we only do imports like this for optional deps like fastBPE